src/DetailsView/Styles/detailsview.scss

```diff
@@ -228,6 +231,70 @@ div.insights-file-issue-details-dialog-container {
         margin-top: 30px;
         font-size: 21px;
     }
+    .footer {
+        margin-bottom: 16px;
+        margin-top: 8px;
+        padding-top: 16px;
+        padding-right: 16px;
+        border-top: #edebe9 1px solid;
+    }
+}
+
+.learn-more {
+    color: $communication-primary;
+    text-decoration: none;
+    display: inline-block;
+    margin-bottom: 24px;
+}
+
+.failure-instance-selector {
+    :first-child {
+        padding-bottom: 8px;
+    }
+}
+
+.failure-instance-selector-note {
+    color: $secondary-text;
+    margin-bottom: 8px;
+}
+
+.failure-instance-snippet-title {
+    color: $primary-text;
+    margin-top: 24px;
+}
+
+.failure-instance-snippet-empty-body {
+    margin-top: 8px;
+    margin-bottom: 24px;
+    color: $secondary-text;
+}
+
+.failure-instance-snippet-filled-body {
+    margin-top: 8px;
+    margin-bottom: 24px;
+    padding-top: 12px;
+    padding-bottom: 12px;
+    padding-left: 16px;
+    padding-right: 16px;
+    max-height: 200px;
+    color: $primary-text;
+    background-color: #f3f3f3;
```

Comment:
Can you please try and use `background-color` from the theme palette in the `colors.scss`. This avoid us having any contrast accessibility issues when the user switch to `high-contrast` theme.